cpu/stm32/Makefile.dep

```diff
@@ -16,10 +16,12 @@ ifneq (,$(filter periph_uart_nonblocking,$(USEMODULE)))
   USEMODULE += tsrb
 endif
 
+ifeq (, $(filter mp1, $(CPU_FAM)))
 ifneq (STM32F030x4, $(CPU_LINE))
```

Comment:
Seems redundant. `CPU_LINE` won't be "STM32F030x4" for stm32mp1.

Reply:
not sure about this, the test is: "ifneq ($(CPU_FAM), mp1)" AND "ifneq (STM32F030x4, $(CPU_LINE))"
I should however change "ifeq (, $(filter mp1, $(CPU_FAM)))" to "ifneq ($(CPU_FAM), mp1)" as you reviewed upper
Do you agree ?

Reply:
Ah sorry I thought it would read `ifneq (STM32F030x4, $(CPU_LINE))`

How about `ifeq (, $(filter mp1, $(CPU_FAM)) $(filter STM32F030x4, $(CPU_LINE)))`?

Reply:
yes better in one line, thx